Foundation/NSFileManager.swift

```diff
@@ -279,8 +285,10 @@ open class FileManager : NSObject {
 
 #if os(OSX) || os(iOS)
         let ti = (TimeInterval(s.st_mtimespec.tv_sec) - kCFAbsoluteTimeIntervalSince1970) + (1.0e-9 * TimeInterval(s.st_mtimespec.tv_nsec))
-#else
+#elseif !os(Android)
```

Comment:
I would prefer `#elseif os(Android)` -- that is, keep the #if in the positive sense.